pkg/helm/release/manager.go

```diff
@@ -144,7 +144,7 @@ func (m *manager) Sync(ctx context.Context) error {
 }
 
 func notFoundErr(err error) bool {
-	return strings.Contains(err.Error(), "not found")
+	return err != nil && strings.Contains(err.Error(), "not found")
```

Comment:
Is there a particular “not found” error type used to express this condition?

Reply:
Unfortunately, no. It comes from here: https://github.com/helm/helm/blob/618447cbf203d147601b4b9bd7f8c37a5d39fbb4/pkg/storage/errors/errors.go#L22

Reply:
Oh fun.